hbase-shell/src/main/ruby/shell/commands/clear_block_cache.rb

```diff
@@ -33,6 +33,7 @@ def help
 
       def command(table_name)
         formatter.row([admin.clear_block_cache(table_name)])
+        nil
```

Comment:
formatter.row returns void? Different to nil?

Reply:
formatter.row returns the running total number of rows printed (`@row_count`) since the last call to formatter.header